code/ui/components/src/components/Button/Button.tsx

```diff
@@ -33,240 +125,64 @@ const ButtonWrapper = styled.button<{
     verticalAlign: 'top',
     whiteSpace: 'nowrap',
     userSelect: 'none',
-    opacity: 1,
+    opacity: disabled ? 0.5 : 1,
```

Comment:
this way of setting something to be disabled is a bit unfortunate. if the button is on top of something that has multiple colors, it will actually be semi-transparent, where I think what we really want is the colors to be slighlty muted?